### DISCUSSION TITLE
整合包搜索支持区分加载器有人交过issues吗

### DISCUSSION THREAD

**Discussion by Chiloven945** [Meloong-Git/PCL#1359] [提问] [answered]
问一下没有提交一个，目前这整合包搜索不支持，我一搜全是forge只有几个fabric这forge优化差的要死（

**Comment by LTCatt** [comment, marked as answer]
CurseForge 整合包搜索不支持区分加载器。它们的 API 不让这样干，所以我也不可能做……
